dependencies/example-projects/grpc-vs-google-api-client/README.md

```diff
@@ -0,0 +1,62 @@
+# grpc 1.17.1 v.s. google-api-client 1.27.0
```

Comment:
I wonder whether we should name these examples for the dependencies or for the case we're demonstrating.

Should this directory be called "missing-method" or something like that? It could still use these two dependencies, but that way if we decide to change the actual dependencies used to something else we wouldn't have a stale directory name.

Reply:
I think we should keep the artifacts involved. We may have more than one example.

Reply:
I used the artifact-specific name so that we document the conflict between the two artifacts with the versions.

This reflects the title of the issue #327 "Document gRPC vs google-api-client".

Reply:
I'm unclear on the point of these examples. Are they meant to be examples of the ways in which static linkage errors could cause runtime errors in user projects? If so, then the specific artifacts involved don't really matter.

If they're meant to be test cases for specific conflicts known to occur, then why aren't they test cases? How does writing them up in markdown help anyone? And if they're meant to be known errors, then they'll get stale as the specific versions shown get older.

Who is the audience for these examples? What do we expect them to learn from them?

Reply:
When filing a ticket to another repository (e.g., google-api-client) to fix a diamond dependency, I would add a link from the ticket to this example directory to explain the problem.

A JUnit test case for `StaticLinkageChecker` is not convincing for that purpose.

Audience would be the readers of such tickets.

Reply:
I think this is critical to prove we're addressing a real problem. These problems aren't hypothetical. They happen, and this is the simplest, most obvious we've found yet. This isn't a test case though. Its documentation and motivation. 

I don't think we should be using personal Github repos for official Github work.